spaces/oceandao/index.json

```
@@ -2,7 +2,7 @@
   "name": "OceanDAO",
   "network": "1",
   "symbol": "OCEAN",
-  "skin": "ocean-dao",
+  "skin": "oceandao",
```

You will need to add domain field to make your subdomain work.
```suggestion
  "skin": "oceandao",
  "domain": "vote.oceanprotocol.com",
```